Data/DynamicInputPack.cs

```diff
@@ -206,6 +206,17 @@ public ICommand EditTextureCommand
 
         private void EditTexture(DynamicInputTexture texture)
         {
+            //Automatically select suitable zoom.
+            if (texture.ScaleFactor == 1)
+            {
+                double scale = (texture.ImageHeight + texture.ImageWidth) / 2;
+                texture.ScaleFactor = 600 / scale;
```

Comment:
Just curious, how did you determine the 600?

Also should it be `600.0`?

Reply:
the value is something like the start display size, we could calculate it on the window size.
decimal places i rarely use does it make a technical difference? or is it just to show that it is a double?

Reply:
No it's fine.  I couldn't remember how the casting works.  I didn't know if you were accidentally casting to an integer here and then saving in a double but it looks like this should be handled properly.